client/src/interface/Notification.ts

```diff
@@ -0,0 +1,19 @@
+export interface Notification {
+  _id: string;
+  user: string;
```

Comment:
You can add optional callback function that will be called when notification is clicked.
This will help to send user to respective page when notification is clicked.